num2words/lang_AR.py

```diff
@@ -39,35 +40,33 @@ def setup(self):
                              "اثناعشر", "أحد عشر", "عشرة", "تسعة", "ثمانية",
                              "سبعة", "ستة", "خمسة", "أربعة", "ثلاثة", "اثنين",
                              "واحد", "صفر"]
-        self.ords = { "واحد"    : "أول",
-                      "اثنين"    : "ثاني",
-                      "ثلاثة"  : "ثالث",
-                      "أربعة": "رابع",
-                      "خمسة"   : "خامس",
-                      "ثمانية"  : "ثامن",
-                      "تسعة"   : "تاسع",
-                      "اثناعشر" : "ثاني عشر" }
-
+        self.ords = {"واحد": "أول",
+                     "اثنين": "ثاني",
+                     "ثلاثة": "ثالث",
+                     "أربعة": "رابع",
+                     "خمسة": "خامس",
+                     "ثمانية": "ثامن",
+                     "تسعة": "تاسع",
+                     "اثناعشر": "ثاني عشر"}
```

Comment:
This should be different, not?

```
self.ords = {
     "واحد" :"أول",
      ....
}
```
The ' ' is after the ':'

Reply:
I not undesrtand your question here, it is about the space around the "," or the identation of new lines?

pycodestyle-3.6 num2words/lang_AR.py 
num2words/lang_AR.py:112:80: E501 line too long (141 > 79 characters)